src/sanitiser.js

```diff
@@ -28,7 +28,9 @@ const sanitiseTestConfiguration = (conf, testId) => {
   configuration.testName = configuration.testNameProvided
     ? conf.testName
     : testConfig.testName;
-  configuration.testId = testId;
+  if (testId !== undefined) {
+    configuration.testId = testId;
+  }
```

Comment:
also add 
```
  configuration.removePostFix = checkProperty(conf, 'removePostFix', 'boolean')
    ? conf.removePostFix
    : globalConfig.saveCurrentImage;
```
To `sanitiseGlobalConfiguration` function.
and 

```
removePostFix: false,
```

to `defaultConfigs.js`